src/main/java/org/gitlab4j/api/models/Commit.java

```diff
@@ -162,6 +163,9 @@ public void setUrl(String url) {
         this.url = url;
     }
 
+    public Pipeline getLastPipeline() { return lastPipeline; }
+
+    public void setLastPipeline(Pipeline lastPipeline) { this.lastPipeline = lastPipeline; }
 
```

Comment:
@Gaff 
To be consistent with the style used in this library change these to multiline methods:
```
public Pipeline getLastPipeline() {
    return lastPipeline;
}

public void setLastPipeline(Pipeline lastPipeline) {
    this.lastPipeline = lastPipeline;
}
```